truss/templates/control/control/helpers/inference_server_process_controller.py

```diff
@@ -52,6 +56,17 @@ def stop(self):
 
         self._inference_server_started = False
 
+    def terminate_with_wait(self):
+        self._inference_server_process.terminate()
+        self._inference_server_terminated = True
+        termination_check_attempts = int(
+            TERMINATION_TIMEOUT_SECS / TERMINATION_CHECK_INTERVAL_SECS
+        )
+        for _ in range(termination_check_attempts):
+            time.sleep(TERMINATION_CHECK_INTERVAL_SECS)
+            if self._inference_server_process.poll() is not None:
```

Comment:
Can we add a link to the [returncode reference](https://docs.python.org/3.9/library/subprocess.html#subprocess.Popen.returncode) for future readability? This wasn't immediately obvious to me